scripts/imgtool/keys/general.py

```diff
@@ -19,17 +19,46 @@ def _public_emit(self, header, trailer, indent, file=sys.stdout, len_format=None
         if len_format is not None:
             print(len_format.format(len(encoded)), file=file)
 
-    def emit_c(self, file=sys.stdout):
+    def _private_emit(self, header, trailer, indent, file=sys.stdout, len_format=None):
+        print(AUTOGEN_MESSAGE, file=file)
+        print(header, end='', file=file)
+        encoded = self.get_private_bytes()
```

Comment:
This looks a bit exaggerated in the copypasta; this line above seems to be the only one changed from `_public_emit`. Why not call it `_emit` or `_emit_from_bytes` and pass in the byte buffer?

Reply:
Done